### ISSUE TITLE
Header pattern: Megamenu doesn't hide when a user's keyboard focus moves onto the main body content.

### ISSUE THREAD

**Issue by jkmnch** [wmcadigital/wmn-design-system#462]
**Describe the bug**
When a user moves keyboard focus from the last link in the Get Help section, the keyboard focus moves onto the link in the body content, but the megamenu is still shown on the screen and doesn't disappear.

**Expected behaviour**
When a user clicks on tab after the last link the header (Report a problem with a shelter or stop), then the megamenu should disappear when the keyboard focus moves to links in the body content.

**Screenshots**
<img width="1792" alt="Screenshot 2020-10-22 at 09 14 45" src="https://user-images.githubusercontent.com/62507489/96845488-ab5afd80-1448-11eb-8822-2e9c6fe7d11e.png">

**Desktop (please complete the following information):**
 - OS: macOS 10.15.7
 - Browser: Chrome
 - Version: 86.0.4240.80

**Additional context**
- There is an issue in Safari where the header turns black when the keyboard focus moves from the last link in the header into the main body content. I'll post this as a separate issue.
